### DISCUSSION TITLE
postgres_fdw cannot connect to external postgres on free tier

### DISCUSSION THREAD

**Discussion by DevKILC** [supabase/supabase#29388] [Questions]
When I try to generate a foreign table, I get a '504 Gateway Time-out' error and cannot retrieve the data. However, when I test it locally or with a self-hosted database (not using Supabase), it works fine.

**Comment by jaydindg** [comment]
I'm getting this same exact issue, have you found a solution?

**Comment by stuartmemo** [comment]
Same here on the Pro plan. Have created a support ticket and will report back.

**Comment by rc-kunal** [comment]
I am facing the same issue; is there any solution/workaround for this?
